analytic_location/model/stock_move.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2015-19 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import models, fields
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _default_analytic_account_id(self):
+        picking_type_id = self.picking_id._context.get(
+            'default_picking_type_id')
+        analytic_account = self.env['account.analytic.account'].search(
+            ['|', ('picking_type_id', '=', picking_type_id),
+             ('picking_type_out_id', '=', picking_type_id),
+             ('parent_id', '=', 1)])
```

Comment:
why this?

Reply:
This is necessary only when in the company all projects are children of a master project, father of all. But as this is a customization better to move to a  custom repo.